engine/reference/commandline/network_create.md

```diff
@@ -160,6 +160,8 @@ equivalent docker daemon flags used for docker0 bridge:
 | `com.docker.network.bridge.host_binding_ipv4`    | `--ip`      | Default IP when binding container ports               |
 | `com.docker.network.driver.mtu`                  | `--mtu`     | Set the containers network MTU                        |
 
+The `com.docker.network.driver.mtu` option is also supported by the `overlay` driver.
```

Comment:
Thanks for the PR. This file is not maintained in `docker.github.io` repo and needs to be changed in `docker/docker` instead. Please remove it from this PR and open another one in that repo. You can link that PR to this one for tracking purposes.

The other file is fine for this PR.